src/Api/InitializeInTracee.cpp

```diff
@@ -4,15 +4,101 @@
 
 #include "Api/InitializeInTracee.h"
 
+#include <absl/base/casts.h>
+#include <absl/container/flat_hash_map.h>
+#include <absl/strings/match.h>
+
+#include <functional>
+
 #include "Api/Orbit.h"
-#include "OrbitBase/Result.h"
+#include "ElfUtils/LinuxMap.h"
+#include "OrbitBase/ExecutablePath.h"
+#include "OrbitBase/Logging.h"
+#include "OrbitBase/UniqueResource.h"
+#include "UserSpaceInstrumentation/Attach.h"
+#include "UserSpaceInstrumentation/ExecuteInProcess.h"
+#include "UserSpaceInstrumentation/InjectLibraryInTracee.h"
 
+using orbit_grpc_protos::ApiFunction;
 using orbit_grpc_protos::CaptureOptions;
+using orbit_grpc_protos::ModuleInfo;
+using orbit_user_space_instrumentation::AttachAndStopProcess;
+using orbit_user_space_instrumentation::DetachAndContinueProcess;
+using orbit_user_space_instrumentation::DlopenInTracee;
+using orbit_user_space_instrumentation::DlsymInTracee;
+using orbit_user_space_instrumentation::ExecuteInProcess;
+
+static ErrorMessageOr<absl::flat_hash_map<std::string, ModuleInfo>> GetModulesByPathForPid(
+    int32_t pid) {
```

Comment:
I use `pid_t` from `#include <sys/types.h>` as a type for `pid`. Probably doesn't matter since it is an int everywhere.

Reply:
yeah, I think it's fine